doc/ref/plotting_options/styling.ipynb

```diff
@@ -0,0 +1,519 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c1ddd044-1771-43a3-a09d-580749371644",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import hvplot.pandas  # noqa\n",
+    "import hvsampledata\n",
+    "\n",
+    "df = hvsampledata.penguins(\"pandas\")\n",
+    "df.hvplot.scatter(\n",
+    "    x='bill_length_mm',\n",
+    "    y='bill_depth_mm',\n",
+    "    by='species',\n",
+    "    fontscale=1.5,\n",
+    "    title=\"Penguins Species\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ee479a25-5270-4b60-91c3-5da24ae2a501",
+   "metadata": {},
+   "source": [
+    "(option-fontsize)=\n",
+    "## `fontsize`\n",
+    "\n",
+    "The `fontsize` option sets the font size for different text elements in the plot. It can be:\n",
+    "- A single value (e.g. 12, '14pt', or '10px') to apply to all text.\n",
+    "- A dictionary to control individual elements like title, axes labels, or ticks.\n",
+    "Example: {'title': '15pt', 'xlabel': '12pt', 'ylabel': '12pt', 'ticks': 10}\n",
+    "\n",
+    "Use this for precise control over text appearance, especially in static reports or dense layouts."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "722250c2-6461-4934-ab19-e8ef3cf3b2e0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import hvplot.pandas  # noqa\n",
+    "import hvsampledata\n",
+    "\n",
+    "df = hvsampledata.penguins(\"pandas\")\n",
+    "\n",
+    "df.hvplot.scatter(\n",
+    "    x='bill_length_mm',\n",
+    "    y='bill_depth_mm',\n",
+    "    by='species',\n",
+    "    fontsize={'title': '16pt', 'xlabel': '12pt', 'ylabel': '14pt', 'ticks': 10},\n",
+    "    title=\"Penguins Measurements by Species\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2b7b0baa-bcc9-4195-b721-84dca8006cbe",
+   "metadata": {},
+   "source": [
+    "(option-rescale_discrete_levels)=\n",
+    "## `rescale_discrete_levels`\n",
+    "\n",
+    "The `rescale_discrete_levels` option improves the visual contrast of discrete values when using `cnorm='eq_hist'`. By default, it adjusts the lower bound of the colormap so that non-zero values appear higher on the scale–helpful when low counts would otherwise appear faded.\n",
+    "\n",
+    "This only has an effect if:\n",
+    "- `cnorm='eq_hist'` is set\n",
+    "- The color values are discrete (e.g., counts or categories)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8f08a504-dde4-435c-acf2-2e77449b402c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import hvplot.pandas  # noqa\n",
+    "import hvsampledata\n",
+    "\n",
+    "df = hvsampledata.earthquakes(\"pandas\")\n",
+    "\n",
+    "# Simulate discrete values by binning magnitude\n",
+    "df['mag_bin'] = df['mag'].round()\n",
+    "\n",
+    "df.hvplot.scatter(\n",
```

Comment:
Please add a plot with `rescale_discrete_levels=False` (True is the default) so people can compare.